pages/common/git-grep.md

```diff
@@ -0,0 +1,19 @@
+# git-grep
+
+> Find strings inside files as they existed anywhere in repo history. Accepts a lot of the same flags as regular grep. Learn more here: https://git-scm.com/docs/git-grep.
+
+- Search for a string in working directory:
+
+`git grep {{search_string}}`
+
+- Search for string in files matching a pattern in working directory:
+
+`git grep {{search_string}} -- {{file_glob_pattern}}`
+
+- Search for a string in working directory, including submodules:
+
+`git grep --recurse-submodules {{search_string}}`
+
+- Search for a string in another point in history:
+
+`git grep {{search_string}} HEAD~2`
```

Comment:
```suggestion
`git grep {{search_string}} {{HEAD~2}}`
```

Reply:
@pxgamer Originally I wrote this as `{{ref_or_commit}}` but revised it to a specific example after reviewing some of the other git- subcommands in this repo, and noting the contributing guides' recommendation not to explain larger concepts. In particular, [here in the git-reset page](https://github.com/tldr-pages/tldr/blob/master/pages/common/git-reset.md) uses `HEAD~2` explicitely rather than explaining the concept of a ref.

I'm happy to accept the change, but wonder what your opinion on the problem of being explicit without constraining the user's idea of what can go there.

Reply:
I haven't heard back so I'm going to accept it for now. It can always be changed later.

Reply:
Sorry, not sure why I missed replying 🤔 That sounds good. 👍